### ISSUE TITLE
Can not paste IMAGES in the body of the email [Bug]

### ISSUE THREAD

**Issue by johanftg** [bkaankose/Wino-Mail#309]
**Describe the bug**
When I am writing an email, I want to to paste images like screenshots, but when I try it, only can see an icon of a broken picture (see the append).

**Expected behavior**
See the image/picture that I pasted.

**Screenshots**
![image](https://github.com/user-attachments/assets/d8a2edd2-12eb-48c1-ad75-32852b81fc55)

**Additional context**
Please, add or fix that! <3

**Comment by Tiktack** [comment]
That's correct. Right now to add images you can use button in "Insert" tab. Paste event is not handled properly right now.

**Comment by johanftg** [comment]
> That's correct. Right now to add images you can use button in "Insert" tab. Paste event is not handled properly right now.

All right. Do you know about a plan to fix or add that feature?

**Comment by Tiktack** [comment]
Yes, I plan to check that soon.

**Comment by Tiktack** [comment]
Unfortunately, issue you found is related to how UWP handles clipboard with Webview2. Issue in their repo exists from 2022 and looks like it won't be fixed, since UWP not in active development. https://github.com/MicrosoftEdge/WebView2Feedback/issues/2900
Also, for the same reason you cannot copy images from Wino Mail and paste it in a different app like Paint. 

Wino Mail planned to be migrated to WinUI3/WinAppSDK in a future, where this should work out of the box. 
So, this issue will be open until migration happens for tracking reasons.

**Comment by bkaankose** [comment]
The blocker is gone after the migration to WinUI3/WinAppSDK with the version v2.x. I can't reproduce the issue anymore.

Labeling as fixed. Please re-open if you still have issues with pasting the images on the composer.
